src/app.pcss

```diff
@@ -1,82 +1,24 @@
-:root {
-	--colors-top: #191919;
-	--colors-ultra-high: #303030;
-	--colors-high: #555;
-	--colors-low: #e7e7e7;
-	--colors-ultra-low: #f9f9f9;
-	--colors-base: #fefefe;
-
-	--colors-dark-top: #fefefe;
-	--colors-dark-ultra-high: #f9f9f9;
-	--colors-dark-high: #e7e7e7;
-	--colors-dark-low: #555;
-	--colors-dark-ultra-low: #303030;
-	--colors-dark-base: #191919;
-
-	--colors-light-top: #191919;
-	--colors-light-ultra-high: #303030;
-	--colors-light-high: #555;
-	--colors-light-low: #e7e7e7;
-	--colors-light-ultra-low: #f9f9f9;
-	--colors-light-base: #fefefe;
-
-	--colors-dark-overlay: rgba(25, 25, 25, 0.7);
-	--colors-light-overlay: rgba(254, 254, 254, 0.7);
-
-	--font-family-sans-serif: sans-serif;
-	--font-family-serif: serif;
-	--font-family-monospace: monospace;
-
-	--font-size-large: 1.5rem;
-	--font-size: 1rem;
-	--font-size-small: 0.75rem;
-
-	--line-height-large: 2rem;
-	--line-height: 1.5rem;
-	--line-height-small: 1rem;
-
-	--letter-spacing-large: 0.03rem;
-	--letter-spacing: 0.02rem;
-	--letter-spacing-small: 0.0375rem;
-
-	--font-size-h1: 3rem;
-	--font-size-h2: 2.5rem;
-	--font-size-h3: 2rem;
-	--font-size-h4: 1.5rem;
-	--font-size-h5: 1rem;
-	--font-size-h6: 0.75rem;
-
-	--font-weight-h1: 700;
-	--font-weight-h2: 700;
-	--font-weight-h3: 700;
-	--font-weight-h4: 700;
-	--font-weight-h5: 700;
-	--font-weight-h6: 700;
-
-	--line-height-h1: 3.5rem;
-	--line-height-h2: 3rem;
-	--line-height-h3: 2.5rem;
-	--line-height-h4: 2rem;
-	--line-height-h5: 1.5rem;
-	--line-height-h6: 1rem;
-
-	--letter-spacing-h4: 0.03rem;
-	--letter-spacing-h5: 0.02rem;
-	--letter-spacing-h6: 0.0375rem;
-
-	--padding: var(--font-size);
-	--double-padding: calc(2 * var(--padding));
-	--half-padding: calc(var(--padding) / 2);
-	--border-radius: 0.25rem;
-}
-
 body {
 	margin: 0;
 	background-color: var(--colors-base);
 }
 a {
 	color: var(--colors-top);
 }
+ol {
+	counter-reset: li;
+	margin-left: 0;
+	padding-left: 0;
+	list-style: none;
+}
+ol li:before {
+	content: counter(li) '. ';
+	color: var(--colors-top);
+	font-family: var(--font-family-sans-serif);
+}
+ol li {
+	counter-increment: li;
+}
```

Comment:
Should this live in global file?

Reply:
Do you mean in `diete.css`? In that case this needs to be improved, because the numbering alignment would break after 9.

I will open an issue about it.

Reply:
No I mean it maybe should live in a component or a page only.